reports/02-emma-universal-basic-income.md

```diff
@@ -28,7 +29,7 @@ URL with opposing evidence of supporting:
 - https://www.latimes.com/local/california/la-pol-ca-basic-income-stockton-reparations-20190415-story.html
 
 #### Follow up
-The main supporters of this policy verify this organization as a key supporter
+Supporting group (UBI Center) did not verify these opposing groups, but suggested candidate opponents that have published critical articles ([The Heritage Foundation](https://www.heritage.org/welfare/report/universal-basic-income-harms-recipients-and-increases-dependence-government), [Center on Budget and Policy Priorities](https://www.cbpp.org/poverty-and-opportunity/commentary-universal-basic-income-may-sound-attractive-but-if-it-occurred)).
```

Comment:
NB: This CBPP article is the same one mentioned above, written by Robert Greenstein.